modules/kmods-installer/README.md

```diff
@@ -0,0 +1,31 @@
+# `kmods-installer` Module for Startingpoint
```

Comment:
My understanding is that this only installs from https://github.com/ublue-os/akmods, so maybe just `akmods` (or maybe `akmods-installer`) would be a better name, since for https://github.com/ublue-os/bling the module name is just `bling`. Usually adding in `-installer`, `-runner`, `-configurator` or something like that is redundant, too.

Reply:
I agree, I will change this